solana/program_v2/programs/gateway_v2/src/instructions/gatekeeper/issue_pass.rs

```diff
@@ -67,7 +67,7 @@ pub struct IssuePass<'info> {
     #[account(
     init,
     payer = payer,
-    space = Pass::ON_CHAIN_SIZE,
```

Comment:
This is awesome... good find

Reply:
The 8 is for the account descriminator... I'm pretty sure I created a constant for this

Reply:
OC_SIZE_DISCRIMINATOR